applications/registration-api/src/main/java/no/dcat/configuration/FdkPermissionEvaluator.java

```diff
@@ -1,26 +1,26 @@
 package no.dcat.configuration;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import no.dcat.service.permission.PermissionsService;
 import org.springframework.security.access.PermissionEvaluator;
 import org.springframework.security.core.Authentication;
-import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.stereotype.Component;
 
 import java.io.Serializable;
 
+@Component
 public class FdkPermissionEvaluator implements PermissionEvaluator {
-    private static final Logger logger = LoggerFactory.getLogger(FdkPermissionEvaluator.class);
 
+    private PermissionsService permissionsService;
```

Comment:
Perhaps I missed something, but is the existence of FdkPermissionEvaluator justified? Isn't its functionality part of PermissionsService?

Reply:
man må registrere en bean som implementerer PermissionEvaluator interface. Det er sikkert mulig at vår service skal implementere den interface, men det blir mer forvirrende, tror jeg.

Reply:
fikset